src/AElf.WebApp.Application.Chain/AppServices/AppTransactionResultService/AppTransactionGetResultService.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading.Tasks;
+using AElf.Kernel.Blockchain.Application;
+using AElf.Kernel.Blockchain.Domain;
+using AElf.Kernel.TransactionPool.Infrastructure;
+using AElf.Types;
+using AElf.WebApp.Application.Chain.Dto;
+using Microsoft.AspNetCore.Routing.Template;
+using Newtonsoft.Json;
+using Volo.Abp.Application.Services;
+using Volo.Abp.DependencyInjection;
+
+namespace AElf.WebApp.Application.Chain.AppServices.AppTransactionResultService
+{
+    public interface IAppTransactionGetResultService 
```

Comment:
every app service should inherited from IApplicationService,
like 
`public interface IBlockChainAppService : IApplicationService`

Reply:
hi:
Here I am following the modification of review above, the uppermost interface is inherited from IApplicationService...
so.the logic inside should not inherit the IApplicationService..

Reply:
Please let me adjust the protection level so that internal functions are not exposed to the presentation layer. : )